### PR TITLE
Add sample for AWS Integrated Application Test Kit

### PR DESCRIPTION
**Pull request by brnkrygs**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python-test-samples/integrated-application-test-kit/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+```bash
+cd ./plugins/plugin_tester && sam build --use-container
+cd ./plugins/plugin_tester && sam deploy --no-confirm-changeset --no-fail-on-empty-changeset
+```
+
+### Run the prevalidate integration tests
+
+The prevalidate plugin is a sample plugin that acts on the [prevalidate event](./events/1-prevalidate-event.md) emitted by the platform. These integration tests use the AWS SDK to query the Amazon CloudFormation API directly and poll the plugin tester workflow.
+
```
Comment (dancfox) [inline python-test-samples/integrated-application-test-kit/README.md:54]: This example shows how you might write tests without the use of the IATK.

File: python-test-samples/integrated-application-test-kit/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Replace the `AWS_REGION` parameter with the region you deployed to.
+
+```bash
+cd ./plugins/1-prevalidate-plugins/test-prevalidate-plugin && npm install
+cd ./plugins/1-prevalidate-plugins/test-prevalidate-plugin && PLUGIN_TESTER_STACK_NAME=video-plugin-tester AWS_REGION=us-west-2 npm test
+```
+
+### Run the postvalidate integration tests
+
+The postvalidate plugin is a sample plugin that acts on the [postvalidate event](./events/2-postvalidate-event.md) emitted by the platform. These integration tests use the Integrated Application Test Kit to find cloud-hosted resources and listen for activity.
```
Comment (dancfox) [inline python-test-samples/integrated-application-test-kit/README.md:73]: In contrast to the previous test example, these tests show how the IATK simplifies some common testing tasks.

File: python-test-samples/integrated-application-test-kit/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+```bash
+cd ./plugins/1-prevalidate-plugins/test-prevalidate-plugin && sam delete
+```
+
+Delete the [plugin tester](./plugins/plugin_tester/)
+
+```bash
+cd ./plugins/plugin_tester && sam delete
+```
```
Comment (dancfox) [inline python-test-samples/integrated-application-test-kit/README.md:112]: To learn more about testing serverless applications go to [serverlessland.com/testing](serverlessland.com/testing).

**Review by dancfox — COMMENTED** [review]
Just a few minor comments. Thank you!

**Comment by brnkrygs** [comment]
Thanks for the review! Updated to address comments.

**Pull request merged**